stripe/src/main/java/com/stripe/android/ApiVersion.kt

```diff
@@ -9,14 +9,36 @@ package com.stripe.android
  * See [https://stripe.com/docs/upgrades](https://stripe.com/docs/upgrades) for latest
  * API changes.
  */
-internal data class ApiVersion internal constructor(internal val code: String) {
+internal data class ApiVersion internal constructor(
+    internal val version: String,
+    internal var betas: Set<StripeApiBeta>? = null
+) {
+    constructor(
+        betas: Set<StripeApiBeta>
+    ) : this(API_VERSION_CODE, betas)
+
+    val code: String
+        get() =
+            betas?.let {
+                listOf(this.version)
+                    .plus(
+                        betas!!.map { it.code }
+                    )
+                    .joinToString(";")
+            } ?: version
 
     override fun toString(): String {
-        return code
+        return betas?.let {
+            listOf(this.version)
+                .plus(
+                    betas!!.map { it.code }
+                )
+                .joinToString(";")
+        } ?: version
```

Comment:
Is this the same thing as `code`? Also I'd be totally fine with removing the `toString` override. Probably unnecessary for a data class.